data/config_options.yml

```
@@ -132,6 +144,9 @@ options:
       since: 0.1.0
       type: string
       default_value: null
+    nodejs:
```

All the Node.js config options need to have a camelcase version instead of the underscored versions. You can add a `config_key` override for every Node.js config option YAML block.

```suggestion
    nodejs:
      config_key: "pushApiKey"
```